fairlearn/postprocessing/_threshold_optimizer.py

```diff
@@ -211,6 +211,27 @@ class ThresholdOptimizer(BaseEstimator, MetaEstimatorMixin):
        Supervised Learning," arXiv.org, 07-Oct-2016.
        [Online]. Available: https://arxiv.org/abs/1610.02413.
 
+    Examples
+    --------
+    >>> from fairlearn.postprocessing import ThresholdOptimizer
+    >>> from sklearn.linear_model import LogisticRegression
+    >>> from sklearn.metrics import balanced_accuracy_score
+    >>> import numpy as np
```

Comment:
If you want to reduce even more you could just create lists (i.e., not even wrap them in `np.array(...)` since it should still work with that. Not really something I will insist on, though.

Reply:
I looked at some other examples (e.g. the `MetricFrame` example and the user guide), and they also used lists. Changed now :).